### PR TITLE
fix(CR): Fixed separate button header for MR and CR at requests page

### PR DESCRIPTION
**Pull request by amritkv**
Added separate button header titles for MR and CR at requests page

### CODE REVIEW

File: src/app/[locale]/requests/components/Requests.tsx [hunk 1]
```diff
@@ -167,6 +171,30 @@ function Requests() {
                 setOpenModerationRequestCount(openMRCount)
                 setClosedModerationRequestCount(closedMRCount)
             })
+        void fetchData('clearingrequests')
+            .then((clearingRequests: EmbeddedClearingRequest) => {
+                let openCRCount = 0
+                let closedCRCount = 0
+                clearingRequests['_embedded']['sw360:clearingRequests']
+                .filter((item: ClearingRequest) => {
+                    if (  item.clearingState === 'NEW' ||
```
Comment (hoangnt2) [inline src/app/[locale]/requests/components/Requests.tsx:180]:
```suggestion
                    if ( item.clearingState === 'NEW' ||
```
Remove redundant space

File: src/app/[locale]/requests/components/Requests.tsx [hunk 2]
```diff
@@ -167,6 +171,30 @@ function Requests() {
...
+                          item.clearingState === 'ACCEPTED' ||
+                          item.clearingState === 'IN_QUEUE' ||
+                          item.clearingState === 'IN_PROGRESS' ||
+                          item.clearingState === 'AWAITING_RESPONSE' ||
+                          item.clearingState === 'ON_HOLD' ||
+                          item.clearingState === 'SANITY_CHECK' ||
+                          item.clearingState === 'PENDING_INPUT'){
+                            openCRCount++;
+                    }
+                    else if ( item.clearingState === 'CLOSED' ||
```
Comment (hoangnt2) [inline src/app/[locale]/requests/components/Requests.tsx:190]:
```suggestion
                    else if (item.clearingState === 'CLOSED' ||
```
Remove redundant space

File: src/app/[locale]/requests/components/Requests.tsx [hunk 3]
```diff
@@ -167,6 +171,30 @@ function Requests() {
...
+                clearingRequests['_embedded']['sw360:clearingRequests']
+                .filter((item: ClearingRequest) => {
+                    if (  item.clearingState === 'NEW' ||
+                          item.clearingState === 'ACCEPTED' ||
+                          item.clearingState === 'IN_QUEUE' ||
+                          item.clearingState === 'IN_PROGRESS' ||
+                          item.clearingState === 'AWAITING_RESPONSE' ||
+                          item.clearingState === 'ON_HOLD' ||
+                          item.clearingState === 'SANITY_CHECK' ||
+                          item.clearingState === 'PENDING_INPUT'){
```
Comment (hoangnt2) [inline src/app/[locale]/requests/components/Requests.tsx:187]:
```suggestion
                          item.clearingState === 'PENDING_INPUT') {
```
Add space before {

File: src/app/[locale]/requests/components/Requests.tsx (hunk 2 above)
Comment (amritkv) [reply src/app/[locale]/requests/components/Requests.tsx]: Fixed !

File: src/app/[locale]/requests/components/Requests.tsx (hunk 3 above)
Comment (amritkv) [reply src/app/[locale]/requests/components/Requests.tsx]: Fixed !

**Pull request merged**